tiny-firmware/firmware/fsm_skycoin_impl.c

```diff
@@ -146,7 +150,7 @@ ErrCode_t msgSkycoinAddressImpl(SkycoinAddress *msg, ResponseSkycoinAddress *res
         return ErrMnemonicRequired;
     }
 
-    if (fsm_getKeyPairAtIndex(msg->address_n, pubkey, seckey, resp, start_index) != ErrOk) {
+    if (fsm_getKeyPairAtIndex(msg->address_n, pubkey, seckey, resp, start_index, address_from_pubkey) != ErrOk) {
```

Comment:
You can omit creating `address_from_pubkey` variable and directly path `&skycoin_address_from_pubkey` to fsm_getKeyPairAtIndex.